views/js/buckaroo.js

```diff
@@ -296,28 +296,28 @@ class BuckarooCheckout {
             parentSelector.children().show();
         } else {
             textElement.text(textElement.attr('text-more'));
-            this.hideExcessIssuers(parentSelector);
+            // this.hideExcessIssuers(parentSelector);
         }
 
         toggle.toggleClass('bk-toggle-down bk-toggle-up');
     }
 
-    hideExcessIssuers(selector) {
-        const isPayByBank = selector.hasClass('bk-paybybank-selector');
-        const selectedIssuer = isPayByBank ? selector.find('input:checked') : null;
-
-        if (isPayByBank && selectedIssuer && selectedIssuer.length) {
-            selector.children().not(selectedIssuer.closest('.bk-method-issuer')).hide();
-        } else {
-            selector.children(`:nth-child(n+${BuckarooCheckout.SHOW_MORE_BANKS})`).hide();
-        }
-    }
+    // hideExcessIssuers(selector) {
```

Comment:
task was asked to remove show more only for ideal and credit card, why you touched paybybank